src/app/components/ms-item/ms-item.component.html

```diff
@@ -6,56 +6,68 @@
-                    <evt-content-viewer [content]="dt"></evt-content-viewer>
-                    <span *ngIf="i < [getDocTitle.length-1]">, </span>
+                <ng-container *ngFor="let docTitles of data.docTitles; let i = index">
+                    <evt-content-viewer *ngFor="let docTitle of docTitles" [content]="docTitle"></evt-content-viewer>
+                    <span *ngIf="i < [data.docTitles.length-1]">, </span>
                 </ng-container>
             </ng-container>
         </evt-ms-desc-section>
         <ng-container class="msItem-information">
-            <evt-ms-desc-section *ngIf="data?.textLang.length>0" [label]="'textLang'" [inlineLabel]="true" [nestedElement]="true"> 
+            <evt-ms-desc-section *ngIf="data?.textLangs?.length>0" [label]="'textLang'" [inlineLabel]="true" [nestedElement]="true"> 
                 <ng-container content>
-                    <ng-container *ngFor="let tl of data.textLang; let i = index">
-                        <evt-content-viewer [content]="tl"></evt-content-viewer>
-                        <span *ngIf="i < [data.textLang.length-1]">, </span>
+                    <ng-container *ngFor="let textLangs of data.textLangs; let i = index">
+                        <evt-content-viewer *ngFor="let textLang of textLangs" [content]="textLang"></evt-content-viewer>
+                        <span *ngIf="i < [data.textLangs.length-1]">, </span>
                     </ng-container>
                 </ng-container>
             </evt-ms-desc-section>
-            <evt-ms-desc-section *ngIf="data?.author.length>0 || data?.docAuthor.length>0" [label]="'author'" [inlineLabel]="true" [nestedElement]="true">
-                <span content>
-                    <ng-container *ngFor="let a of getAuthor; let i = index">
-                        <evt-content-viewer [content]="a"></evt-content-viewer>
-                        <span *ngIf="i < [getAuthor.length-1]">, </span>
-                    </ng-container>
-                    <span *ngIf="data?.author.length>0 && data?.docAuthor.length>0">, </span>
-                    <ng-container *ngFor="let da of getDocAuthor; let i = index">
-                        <evt-content-viewer [content]="da"></evt-content-viewer>
-                        <span *ngIf="i < [getDocAuthor.length-1]">, </span>
-                    </ng-container>
-                </span>
+            <evt-ms-desc-section *ngIf="data?.authors?.length>0 || data?.docAuthors?.length>0" [label]="'author'" [inlineLabel]="true" [nestedElement]="true">
+                <ng-container content>
+                    <span *ngFor="let authors of data?.authors; let i = index">
+                        <evt-content-viewer *ngFor="let author of authors" [content]="author"></evt-content-viewer>
+                        <span *ngIf="i < [data.authors.length-1]">, </span>
+                    </span>
+                    <span *ngIf="data?.authors.length>0 && data?.docAuthors.length>0">, </span>
```

Comment:
I think you don't need the "?" here on data, because you have already checked above